tests/run/cpp_stl_random.pyx

```diff
@@ -1,12 +1,13 @@
 # mode: run
 # tag: cpp, cpp11
 
+from __future__ import print_function
```

Comment:
I'm not sure if this global import helps in doctests. But you're also not making use of it in any of the `print()` calls below, so…

Reply:
Ah, sorry, this line tells python2 to use python3 syntax for print, i.e. `print(some, values)` rather than `print some, values`. Admittedly, importing an object that's later unused to change the code parsing is a somewhat surprising choice. Some more details [here](https://docs.python.org/3/reference/simple_stmts.html#future).

Reply:
:-D  Thanks, but I know what future imports do. I implemented that specific one in Cython back in 2008.

I was suggesting that a) I don't think this import has an effect on doctests in strings of the same module (but I don't know for sure), and b) the print calls below work exactly the same as print statements in Py2, even without the future import. So, in any case, the future import is not needed here.

Reply:
Ha, I'm of course preaching to the choir, apologies. I'll drop the future import, and it looks like the tests are passing.

Reply:
Ok, so, I looked up the code in [doctest](https://github.com/python/cpython/blob/main/Lib/doctest.py) and it has support for reading the future imports used in the module (good to know, and certainly makes sense). However, this is a compiled Cython module, and `doctest` certainly cannot figure out the original future imports used in the source code there.